azurelinuxagent/common/protocol/restapi.py

```diff
@@ -85,13 +85,15 @@ def __init__(self,  # pylint: disable=R0913
                  publicSettings=None,
                  protectedSettings=None,
                  certificateThumbprint=None,
-                 dependencyLevel=0):
+                 dependencyLevel=0,
+                 state="enabled"):
         self.name = name
         self.sequenceNumber = sequenceNumber  # pylint: disable=C0103
         self.publicSettings = publicSettings  # pylint: disable=C0103
         self.protectedSettings = protectedSettings  # pylint: disable=C0103
         self.certificateThumbprint = certificateThumbprint  # pylint: disable=C0103
         self.dependencyLevel = dependencyLevel  # pylint: disable=C0103
+        self.state = state
```

Comment:
For MultiConfig, each settings also has a state (as compared to before where the state was only available in Plugin level)